collections/_article/dev-snapshot-godot-4-2-beta-3.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Dev snapshot: Godot 4.2 beta 3"
+excerpt: "We continue iterating quickly on beta snapshots for Godot 4.2, to ensure that we can solve regressions before the stable release."
```

Comment:
```suggestion
excerpt: "We continue iterating quickly on beta snapshots for Godot 4.2 to ensure that we can solve regressions before the stable release."
```

Reply:
I think this was to emphasize the second part, maybe a dash is more appropriate?

Reply:
I think both are fine, so I'm good removing the comma.